recipes/timemory/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "timemory" %}
+{% set version = "3.2.1rc10" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7c46677215f66650b344e7980ce91605858dc7505835ab86a11520786a240a67
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt -j 2 \
+    --cxx-standard=17 \
+    --enable-portable \
+    --enable-static-libs \
+    --disable-c \
+    --disable-fortran \
+    --disable-ert \
+    --disable-develop \
+    --disable-shared-libs \
+    --disable-unity-build \
+    --disable-require-packages \
+    --disable-ompt \
+    --disable-upcxx \
+    --disable-kokkos \
+    --disable-caliper \
+    --disable-dyninst \
+    --disable-gperftools \
+    --disable-python-hatchet \
+    --disable-compiler-instrumentation"
+  skip: true  # [osx or (win and vc<15)]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ mpi }}  # [mpi != 'nompi']
+    - cmake
+  host:
+    - python
+    - pip
+    - cython
+    - pybind11
+    - pybind11-abi
+    - scikit-build
+    - {{ mpi }}  # [mpi != 'nompi']
+  run:
+    - python
+    - six
+    - numpy
+    - pydot
+    - pillow
+    - pandas
+    - matplotlib
```

Do you need the Qt backend? Otherwise, `-base` is the right choice.
```suggestion
    - matplotlib-base
```